scripts/install

```diff
@@ -38,76 +42,76 @@ grep -q -R 'jessie-backports' /etc/apt/sources.list{,.d} || {
 }
 
 # Install dependencies
-ynh_package_install_from_equivs ../conf/${DEPS_PKG_NAME}.control \
-  || ynh_die "Unable to install dependencies"
+ynh_package_install_from_equivs ../conf/${DEPS_PKG_NAME}.control
```

Comment:
No you don't need all that, see an example [here](https://github.com/YunoHost-Apps/wallabag2_ynh/blob/master/scripts/remove#L30).